Add Multiline Text Field (#63)
saritasa-nest/saritasa-forge-admin: README.md

{
    builder.SetTruncationMaxCharacters(25);
});
```

# Multiline Text Field property

> You can mark a property as multiline text field. Is allows inpur text in the several text rows. Disabled by default.

Reviewer: ```suggestion
You can mark a property as multiline text field. It allows input of text in several text rows. Disabled by default.
```

## Configuration

### Using Fluent API

```csharp
entityOptionsBuilder.ConfigureProperty(address => address.Street, builder =>
{
    builder.SetIsMultiline(true);
});
```

### Using Attribute

```csharp
[NetForgeProperty(IsMultiline = true)]
public required string Street { get; set; }
```

> Also, there are some auxalary additional properties that are used in the multiple text field

Reviewer 2: auxiliary?
```suggestion
Also, there are some auxiliary properties that are used in the multiple text field.
```

## Lines property is the number of lines of the multiline text field. This is equal to 5 by default.
### Using Fluent API

```csharp
entityOptionsBuilder.ConfigureProperty(address => address.Street, builder =>
{
    builder.SetIsMultiline(true);
    builder.SetLines(15); // sets the lines value as 15
});
```

### Using Attribute

```csharp
[NetForgeProperty(IsMultiline = true, Lines = 15)]
public required string Street { get; set; }
```

## MaxLines property is the max. number of lines of the multiline text field. This is equal to 5 by default.
### Using Fluent API

```csharp
entityOptionsBuilder.ConfigureProperty(address => address.Street, builder =>
{
    builder.SetIsMultiline(true);
    builder.SetMaxLines(15); // sets the max lines value as 15
});
```

### Using Attribute

```csharp

> [NetForgeProperty(IsMultiline = true, MaxLines = 15)]

Reviewer: Let's create a separate attribute like `MultilineFieldAttribute`. with `lines` , `maxLines`, `isAutoGrow` properties.

public required string Street { get; set; }
```

## IsAutoGrow property identifies whether the height of the text field automatically changes with the number of lines of text.
### Using Fluent API

```csharp
entityOptionsBuilder.ConfigureProperty(address => address.Street, builder =>
{

>     builder.SetIsMultiline(true);

Reviewer: What if we'll combine all these configurations in a single method?

Like: `optionsBuilder.SetIsMultiline(autoGrow: false, lines: 5, maxLines: 10);`?
